infrastructure/main.tf

```diff
@@ -71,6 +71,12 @@ data "azurerm_key_vault_secret" "idam_service_key" {
   key_vault_id = "${data.azurerm_key_vault.s2s_vault.id}"
 }
 
+resource azurerm_key_vault_secret "idam_service_secret" {
+  name = "microservicekey-ccd-gw"
```

Comment:
Terraform change needs to go in first PR for it to work as it will look for secret and not find it in PR